<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PontoDigital</class>
 <widget class="QDialog" name="PontoDigital">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Relógio de ponto</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 205, 210);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-style: outset;
border-width:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>Digite seu login</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(255, 255, 255);
border-style: outset;
border-width:2px;
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>Digite sua senha</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="icone_cadeado">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>icon_cadeado.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="icone_usuario">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>icon_usuario.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cadastrar_novo_usuario">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 90, 110);

</string>
   </property>
   <property name="text">
    <string>Cadastrar usuário</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>401</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-width:2px;
border-radius: 5px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Ponto Digital</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_entrada">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(220, 90, 110);
</string>
   </property>
   <property name="text">
    <string>Entrada</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_saida">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(200, 100, 150);

</string>
   </property>
   <property name="text">
    <string>Saida</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_pesquisar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>410</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 90, 110);
</string>
   </property>
   <property name="text">
    <string>Consultar marcações</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_admin">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Login Admin</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>button_entrada</tabstop>
  <tabstop>button_saida</tabstop>
  <tabstop>button_cadastrar_novo_usuario</tabstop>
  <tabstop>button_pesquisar</tabstop>
  <tabstop>calendarWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
